config/jobs/kubernetes-sigs/kind/kind-postsubmits.yaml

```diff
@@ -17,6 +18,13 @@ postsubmits:
         # we need privileged mode in order to do docker in docker
         securityContext:
           privileged: true
+        resources:
+          limits:
+            cpu: 2
```

Comment:
seems pretty low for unit testing?
also kind doesn't have the hack to enable GOMAXPROCS autodetection so we need to consider this when moving unit tests

Reply:
Happy to bump it up. Do you have any recommendations?